app/src/main/java/com/example/cinemapp/data/ImagesResponseDTO.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.cinemapp.data
+
+import com.google.gson.annotations.SerializedName
+
+data class ImagesResponseDTO(
```

Comment:
Should that be changed in every DTO class?

Reply:
Yes, default values are not needed for DTO classes, since calling an API would populate the fields with values from the response and depending on library, null by default if the field is not found, which would only happen if the api changes its response